modules/game_console/console.otui

```diff
@@ -66,7 +66,7 @@ Panel
     anchors.top: parent.top
     margin-left: 13
     margin-top: 8
-    @onCheckChange: updateChatMode()
+    @onCheckChange: toggleChat()
```

Comment:
Take into account the meaning of the functions name...
In your case, toggleChat doesn't really toggle anything.